src/components/connection-prompt/ConnectDialogContainer.svelte

```diff
@@ -82,6 +83,10 @@
           $connectionDialogState.connectionState = ConnectDialogStates.USB_TRY_AGAIN;
           usbTryAgainType = 'close tabs';
           break;
+        } else if (/Only V2 is supported/.test(err.message)) {
```

Comment:
Let's find a better way to identify this case.

Reply:
So the error is thrown here: https://github.com/microbit-foundation/ml-trainer/blob/fea374478d5cb7af929e040a9c9eb63c04030cb7/src/script/microbit-interfacing/Microbits.ts#L47

This is where the version is found: https://github.com/microbit-foundation/ml-trainer/blob/fea374478d5cb7af929e040a9c9eb63c04030cb7/src/script/microbit-interfacing/Microbits.ts#L893

So I suppose I can make a new class method that returns the version and then check the version earlier so that I don't need to catch the error to find it out? 🤔  Ok, will try that

Reply:
Woops I realise the code I was looking at wasn't in main. Will update comment above